### PR TITLE
new user input: disable_index_while_building

### PR DESCRIPTION
**Pull request by BirmacherAkos**
Indexing is needed for

* Autocomplete
* Ability to quickly jump to definition
* Get class and method help by alt clicking.

 Which are not needed in CI environment.

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -142,7 +142,7 @@ workflows:
   ci-parallel_headless_mode:
     envs:
     - SAMPLE_APP_URL: https://github.com/bitrise-samples/sample-swift-project-with-parallel-ui-test.git
-    - SAMPLE_APP_BRANCH: master
+    - SAMPLE_APP_BRANCH: swift_4
```
Comment (BirmacherAkos) [inline bitrise.yml]: Temporary solution. 
Discuss how to fix the Swift version issues.

File: step.yml [hunk 2]
```diff
@@ -111,6 +111,23 @@ inputs:
         - "yes"
         - "no"
       is_required: true
+  - disable_index_while_building: "yes"
+    opts:
+      title: "Disable index while building"
```
Comment (trapacska) [inline step.yml]: `Disable indexing during the build` would sound better I guess.

File: step.yml [hunk 3]
```diff
@@ -111,6 +111,23 @@ inputs:
...
+      description: |-
+        Disable indexing during the build.
+
+        Indexing is needed for
+        
+        * Autocomplete
+        * Ability to quickly jump to definition
+        * Get class and method help by alt clicking.
+
+        Which are not needed in CI environment.
```
Comment (trapacska) [inline step.yml]: Please clarify that this is a build in xcode feature and not something in out step.

Comment (BirmacherAkos) [reply step.yml]: Fixed in:
https://github.com/bitrise-steplib/steps-xcode-test/pull/111/commits/dbcded918e7354f29d6051ce30f3376994436422

File: step.yml (hunk 2 above)
Comment (BirmacherAkos) [reply step.yml]: Fixed in:
https://github.com/bitrise-steplib/steps-xcode-test/pull/111/commits/dbcded918e7354f29d6051ce30f3376994436422

**Pull request merged**